src/include/catalog/storage_pending_deletes_redo.h

```diff
@@ -0,0 +1,32 @@
+/*-------------------------------------------------------------------------
+ *
+ * storage_pending_deletes_redo.h
+ *	  prototypes for functions in backend/catalog/storage_pending_deletes_redo.c
+ *
+ * Copyright (c) 2025 Greengage Community
+ *
+ * IDENTIFICATION
+ * src/include/catalog/storage_pending_deletes_redo.h
+ *
+ *-------------------------------------------------------------------------
+ */
+#ifndef STORAGE_PENDING_DELETES_REDO_H
+#define STORAGE_PENDING_DELETES_REDO_H
+
+#include "postgres.h"
+
+#include "access/xlog.h"
+#include "catalog/storage_pending_deletes.h"
+
+extern void PdlXLogInsert(void);
+
+extern void PdlRedoAdd(PendingRelXactDelete * pd);
+
+extern void PdlRedoXLogRecord(XLogRecord *record);
+
+extern void PdlRedoRemoveTree(TransactionId xid,
+				  TransactionId *sub_xids, int nsubxacts);
+
+extern void PdlRedoDropFiles(void);
```

Comment:
We can declare a function without `extern`. It will be the same declaration.
```suggestion
void PdlXLogInsert(void);

void PdlRedoAdd(PendingRelXactDelete * pd);

void PdlRedoXLogRecord(XLogRecord *record);

void PdlRedoRemoveTree(TransactionId xid,
				  TransactionId *sub_xids, int nsubxacts);

void PdlRedoDropFiles(void);
```

Reply:
Neighbor files like https://github.com/arenadata/gpdb/blob/75ff739d646588018466335eeefb015248591ecb/src/include/catalog/storage.h, https://github.com/arenadata/gpdb/blob/75ff739d646588018466335eeefb015248591ecb/src/include/catalog/storage_database.h, https://github.com/arenadata/gpdb/blob/75ff739d646588018466335eeefb015248591ecb/src/include/catalog/storage_xlog.h all have `external`, so I kept the new file consistent with others. I suggest to keep it as is.